g_docs.go

```diff
@@ -701,143 +742,33 @@ func getModel(str string) (pkgpath, objectname string, m swagger.Schema, realTyp
 	if len(rootapi.Definitions) == 0 {
 		rootapi.Definitions = make(map[string]swagger.Schema)
 	}
+	objectname = objectWithPackageName(objectname, packageName)
 	rootapi.Definitions[objectname] = m
 	return
 }
 
-func parseObject(d *ast.Object, k string, m *swagger.Schema, realTypes *[]string, astPkgs map[string]*ast.Package) {
-	ts, ok := d.Decl.(*ast.TypeSpec)
+func (res *objectResource) parseObject() {
```

Comment:
Nitpicking here but now this is a method instead of a function, we could simply name it `parse` (and return an error as per previous comment)